frame/contracts/src/wasm/mod.rs

```diff
@@ -295,6 +295,7 @@ mod tests {
 		schedule: Schedule<Test>,
 		gas_meter: GasMeter<Test>,
 		debug_buffer: Vec<u8>,
+		ecdsa_recover: Vec<([u8; 65], [u8; 32])>,
```

Comment:
Now since `ecda_recover` isn't mutable anymore you need to use a `RefCell` here.